src/main/java/org/broadinstitute/hellbender/engine/filters/AmbiguousBaseReadFilter.java

```diff
@@ -7,38 +7,42 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Filters out reads with more than a threshold number of N's
+ * Filters out reads with more than a threshold number of N's. By default will determine the threshold by the given
+ * maximum fraction of bases in the read. This can be overridden by setting the maximum number of ambiguous bases instead.
  */
 @DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
 public final class AmbiguousBaseReadFilter extends ReadFilter {
 
     private static final long serialVersionUID = 1L;
+    private final Integer maxAmbiguousBases;
```

Comment:
@magicDGS Thanks for your suggestion - I do see how this could be confusing for a user.

In PathSeq, I have a `--filterMaxAmbiguousBases` argument, but it does something fundamentally different from the default read filter because it applies the filter after masking the reads with more N's, whereas default read filters are applied directly to the input. 

For my tool, I will rename my argument `--maxMaskedBases` to be clearer that this filter occurs after applying masking to the reads. Also, I'll print a warning message if the user tried to use `--ambigFilterFrac` or `--ambigFilterBases`, in case they mistook these for the tool's argument.

I also took your suggestion to add the number of bases as an argument, which I named `--ambigFilterBases` for consistency.